src/codebook/controls/filters/update.js

```diff
@@ -0,0 +1,75 @@
+/*------------------------------------------------------------------------------------------------\
```

Comment:
Still have to call enter()/exit() from update() right?  Just committed a second go.